x/emissions/keeper/genesis.go

```diff
@@ -1007,6 +1007,35 @@ func (k *Keeper) InitGenesis(ctx context.Context, data *types.GenesisState) erro
 			}
 		}
 	}
+
+	// MonthlyReputerRewards
+	if data.MonthlyReputerRewards.GT(cosmosMath.ZeroInt()) {
+		if err := types.ValidateSdkIntRepresentingMonetaryValue(data.MonthlyReputerRewards); err != nil {
+			return errors.Wrap(err, "monthly reputer rewards validation failed")
+		}
+		if err := k.SetMonthlyReputerRewards(ctx, data.MonthlyReputerRewards); err != nil {
+			return errors.Wrap(err, "error setting monthlyReputerRewards")
+		}
+	} else {
+		if err := k.SetMonthlyReputerRewards(ctx, cosmosMath.ZeroInt()); err != nil {
+			return errors.Wrap(err, "error setting monthlyReputerRewards to zero int")
+		}
+	}
```

Comment:
```suggestion
	// MonthlyReputerRewards
	if err := types.ValidateSdkIntRepresentingMonetaryValue(data.MonthlyReputerRewards); err != nil {
		return errors.Wrap(err, "monthly reputer rewards validation failed")
	}
	if err := k.SetMonthlyReputerRewards(ctx, data.MonthlyReputerRewards); err != nil {
		return errors.Wrap(err, "error setting monthlyReputerRewards")
	}
```

This should be enough I guess, same simplification can apply for the `MonthlyTopicRewards`